README.md

```diff
@@ -17,6 +17,7 @@ Algorithms that shine in this setting in terms of both model size and compute, n
  - **EMI-RNN**: Training routine to recover the critical signature from time series data for faster and accurate RNN predictions.
  - **Shallow RNN**: A meta-architecture for training RNNs that can be applied to streaming data.
  - **FastRNN & FastGRNN - FastCells**: **F**ast, **A**ccurate, **S**table and **T**iny (**G**ated) RNN cells.
+ - **Conv1D**: 1-D regular and low-rank convolution architectures for time-series data.
```

Comment:
I dont think this is a major change that merits update on README yet. Lets hold off on this until we release a full audio pipeline.

Reply:
I agree, please revert this change.

Reply:
I've reverted the change